handlers/eth1Blocks.go

```diff
@@ -215,8 +216,7 @@ func getEth1BlocksTableData(draw, start, length, recordsTotal uint64) (*types.Da
 		burned := new(big.Int).Mul(baseFee, big.NewInt(int64(b.GetGasUsed())))
 		burnedPercentage := float64(0.0)
 		if len(txReward.Bits()) != 0 {
-			txBurnedBig := new(big.Float).SetInt(burned)
-			txBurnedBig.Quo(txBurnedBig, new(big.Float).SetInt(txReward))
+			txBurnedBig := decimal.NewFromBigInt(burned, 0).DivRound(decimal.NewFromBigInt(txReward, 0), 18)
```

Comment:
This division is supposed to calculate the ratio of burned fees to the tx reward fees, if that's what you're asking? The reason why I'm using a precision of 18 is because I presume both values to be in Wei, so I don't want to lose anything when dividing. If you think it's unnecessary please let me know.

Reply:
While 18 doesn't guarantee you that no precision is lost it is surely enough for a float64 conversion immediately after it.
Let's leave it like this.